### PR TITLE
Fix host plugin run to remove --port arg of theia run

### PR DESCRIPTION
**Pull request by amiramw**
Signed-off-by: Amiram Wingarten <amiram.wingarten@sap.com>

Remove `--port 4444` in addition to the previously removal support for `--port=4444`

Fixes: https://github.com/theia-ide/theia/issues/4361

### CODE REVIEW

File: packages/plugin-ext/src/hosted/node/hosted-instance-manager.ts [hunk 1]
```diff
@@ -14,22 +14,23 @@
...
 import URI from '@theia/core/lib/common/uri';
-import { ContributionProvider } from '@theia/core/lib/common/contribution-provider';
-import { LogType } from './../../common/types';
-import { HostedPluginUriPostProcessor, HostedPluginUriPostProcessorSymbolName } from './hosted-plugin-uri-postprocessor';
-import { HostedPluginSupport } from './hosted-plugin';
-import { DebugConfiguration } from '../../common';
-import { environment } from '@theia/core';
-import { MetadataScanner } from './metadata-scanner';
-import { FileUri } from '@theia/core/lib/node/file-uri';
+import {ContributionProvider} from '@theia/core/lib/common/contribution-provider';
```
Comment (benoitf) [inline packages/plugin-ext/src/hosted/node/hosted-instance-manager.ts]: hello, could you please keep the formatting options ?

File: packages/plugin-ext/src/hosted/node/hosted-instance-manager.ts [hunk 2]
```diff
@@ -244,9 +245,9 @@ export abstract class AbstractHostedInstanceManager implements HostedInstanceMan
         if (environment.electron.is()) {
             command = ['yarn', 'theia', 'start'];
         } else {
-            command = processArguments.filter(arg => {
-                // remove --port= argument if set
-                if (arg.startsWith('--port=')) {
+            command = processArguments.filter((arg, index, args) => {
+                // remove --port=X and --port X arguments if set
+                if (arg.startsWith('--port') || args[index - 1] === '--port') {
```
Comment (benoitf) [inline packages/plugin-ext/src/hosted/node/hosted-instance-manager.ts]: thx for the fix. But it seems only this part is modified but there are extra 100 changes that are not related to the fix (wrong format of code), could you please cleanup your PR ?

Comment (amiramw) [reply packages/plugin-ext/src/hosted/node/hosted-instance-manager.ts]: Sorry, didn't notice. I will fix it.

Comment (amiramw) [reply packages/plugin-ext/src/hosted/node/hosted-instance-manager.ts]: @benoitf now the formatting is the same as before

**Comment by benoitf** [comment]
There is a valid ECA on file for amiram.winga***@***.com
